R/resampleClusteredRegions.R

```diff
@@ -100,10 +101,8 @@ resampleClusteredRegions <- function(voxel_df_long, n_pca = 20,
 
     # append to cluster_vec
     cluster_vec[, i] <- clusters
-
     message(paste0("Cluster ", i, " calculated"))
```

Comment:
"Cluster" is an incorrect term. What is being calculated? Try to make the messages a bit more informative for the user.